### ISSUE TITLE
indexed httpsOnly not possible

### ISSUE THREAD

**Issue by basz** [vfarcic/docker-flow-proxy#297]
The following does not create a redirect rule for the indexed service domain.

```
labels:
   - com.df.httpsOnly.2=true
```

does it for all configured service domains, which may not be what you want.

```
   - com.df.httpsOnly=true
```

**Comment by vfarcic** [comment]
At the moment, `httpsOnly` is not indexed so it is applied to the whole service. I'll make it support indexes soon (sometimes during the next week).

**Comment by vfarcic** [comment]
It's available in the [release 17.07.24-10](https://github.com/vfarcic/docker-flow-proxy/releases/tag/17.07.24-10). Please try it out and let me know if it works as expected.
